jaxley/modules/base.py

```diff
@@ -41,6 +42,23 @@
 from jaxley.utils.swc import build_radiuses_from_xyzr
 
 
+def only_allow_module(func):
```

Comment:
I added a decorator that we can add to functions that cannot be run on Views (yet).
I hope I did not miss any.

Reply:
Great!